contrib/docker/testnet.sh

```diff
@@ -278,15 +322,39 @@ catch()
   fi
 }
 
+# Read handler
+# $1 - message
+# $2 - varname to set
+# $3 - function or string to execute
+read_input()
+{
+  read -r -p "$1 [Y/n] " REPLY
+  case $REPLY in
+    [nN])
+      ;;
+    *)
+      if [[ $2 != "NULL" ]]; then  # hack to ensure 2nd arg is unused
+        read -r -p "Set new: " REPLY
+        eval ${2}=${REPLY}
```

Comment:
Could use a catch here.
I made the mistake to not put quotes for KOVRI_BIN_ARGS and i didn't see the error

Reply:
> I made the mistake to not put quotes for KOVRI_BIN_ARGS and i didn't see the error

- IIRC, there's never an instance in bash where you would *not* quote a space if you don't want to interpret a space (with the exception of escaping)
- The user-input string shows to quote the arguments as a string

>Could use a catch here.

Instead, I think we should add `eval ${2}=\"${REPLY}\"` to avoid the error all-together. Catching within this handler would require a generic message which could be as cryptic as the shell's error message.

Reply:
:+1: